changelog/rework_format_docs.dd

```diff
@@ -0,0 +1,17 @@
+Documentation of `std.format` has been completely reworked.
+
+In the last years, the documentation of `std.format` was outdated
+little by little and therefore needed a complete rework. The whole
+package was reviewed and all documentations, including examples,
+improved and extended.
+
+Some highlights:
+
+$(UL
+$(LI The grammar of the format string was updated.)
+$(LI A detailed description of format specifiers was provided.)
+$(LI Several examples on how to use the functions and the format
+     strings were added.)
+$(LI The first example(s) of each item were reworked to match the
+     needs of inexperienced programmers.)
```

Comment:
I would rephrase or even drop this last item. If you want to have this in the changelog it should not contain negative attributes like "inexperienced". Something along the lines of: "The examples for each function have been reworked to provide a smooth transition from simple, yet comprehensive, use cases to more complex and difficult scenarios."

Reply:
Dropped it. It's not that important. :-)